### PR TITLE
p5js data + test page fix

### LINKED ISSUE #255: Migrate from processing.js to p5js
Processing.js is no longer being developed and new bugs are not fixed. It was superseded by p5js and we should change quill to use p5js under the hood.

### PR DESCRIPTION
**Pull request by anthonygalea**
#255

### CODE REVIEW

File: src/cljc/quil/core.cljc [hunk 1]
```diff
@@ -682,22 +682,21 @@
...
-    :category "Data"
-    :subcategory "Conversion"
-    :added "1.0"}
-  binary
-  "Returns a string representing the binary value of an int, char or
-  byte. When converting an int to a string, it is possible to specify
-  the number of digits used."
-  ([val]
-   #?(:clj (PApplet/binary (int val))
-      :cljs (.binary (ap/current-applet) val)))
```
Comment (nbeloglazov) [inline src/cljc/quil/core.cljc]: We can use JS to implement it:

`(.toString val 2)`

and 

`(subs (.toString val 2) 0 num-digits)`

for the second one

File: src/cljc/quil/core.cljc [hunk 2]
```diff
@@ -4367,18 +4366,18 @@
...
-    :processing-name "unbinary()"
-    :category "Data"
-    :subcategory "Conversion"
-    :added "1.0"}
-  unbinary
-  "Unpack a binary string to an integer. See binary for converting
-  integers to strings."
-  [str-val]
-  #?(:clj (PApplet/unbinary (str str-val))
-     :cljs (.unbinary (ap/current-applet) (str str-val))))
```
Comment (nbeloglazov) [inline src/cljc/quil/core.cljc]: Same here, can use `(js/parseInt str-val 2)` to parse from binary.

Comment (anthonygalea) [reply src/cljc/quil/core.cljc]: 👍

File: src/cljc/quil/core.cljc (hunk 1 above)
Comment (anthonygalea) [reply src/cljc/quil/core.cljc]: 👍

**Pull request merged**